### DISCUSSION TITLE
Use the stable Actix Web 4.

### DISCUSSION THREAD

**Discussion by afiqzx** [framesurge/perseus#170] [General]
Is there any reason why perseus-actix-web is stuck in 4.0.0-rc.3?

Are you waiting to stabilize the feature for perseus 0.4 first?

I'm willing to work on this, but I want to know what to check/test before pushing any changes.

**Comment by arctic-hen7** [comment]
That is a superb question! Probably because I just haven't updated the dependency yet...

If you'd like to, you're more than welcome to fix this in a PR, and that can go out in the next beta of v0.4.0.

**Reply by afiqzx** [reply]
Yeah I'll try to do it over the weekend. Should I use actix-web 4.0 or 4.1.0?

**Reply by arctic-hen7** [reply]
The latest stable version should be fine. All the examples are tested on CI with Actix Web, so as long as everything compiles locally and runs on CI we should be fine (famous last words).
